tests/qeidentity/common/tests.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/report.h>
+
+#ifdef OE_BUILD_ENCLAVE
+
+#define Memset oe_memset
+#define Memcpy oe_memcpy
+#define Memcmp oe_memcmp
+
+#define GetReport oe_get_report
+
+#define VerifyReport oe_verify_report
+
+#define TEST_FCN OE_ECALL
+
+#else
+
+#define Memset memset
+#define Memcpy memcpy
+#define Memcmp memcmp
+
+#define TEST_FCN
+
+#endif
+
+#define OE_LOCAL_REPORT_SIZE (sizeof(oe_report_header_t) + sizeof(sgx_report_t))
+
+/*
+ * g_unique_id is populated from the first call to oe_parse_report.
+ * The enclave's unique_id is asserted to not change subsequently.
+ */
+uint8_t g_unique_id[32];
+
+uint8_t g_signer_id[32] = {0xca, 0x9a, 0xd7, 0x33, 0x14, 0x48, 0x98, 0x0a,
```

Comment:
This entire file does not seem to be used.
I can comment out these variable declarations and still the test builds fine.